digitalstrom_mqtt/mqtt.go

```diff
@@ -194,6 +224,254 @@ func (dm *DigitalstromMqtt) getTopic(deviceType string, deviceId string, deviceN
 	return topic
 }
 
+// Returns MQTT topic to publish the Server status.
+func (dm *DigitalstromMqtt) getStatusTopic() string {
+	root_topic := strings.Split(dm.config.TopicFormat, "/")[0]
```

Comment:
This is not good and will leave only confusion. It's on me for willing to have a highly customizable topic... 

I'm not sure if people are using this feature anyway.

Please put a  `FIXME:  use a topic prefix` on this function. I will come back to it later.

Reply:
Done. Indeed I found it quite confusing that you could customise the whole message tree. It is common on other projects to customise only the root topic where to publish and the rest is up to each project to set the topic structure.

I would have change it to use a topic prefix, but I though that this should not be my call.